storm-server/src/main/java/org/apache/storm/LocalCluster.java

```diff
@@ -196,6 +196,9 @@ private LocalCluster(Builder builder) throws Exception {
             this.tmpDirs.add(nimbusTmp);
             stormHomeBackup = System.getProperty(ConfigUtils.STORM_HOME);
             TmpPath stormHome = new TmpPath();
+            if (!stormHome.getFile().mkdirs()) {
```

Comment:
I might be going to too strict, but this looks like follow-up of STORM-3116 (not relevant to Time) which would be better to have individual PR. The change is minor so no need to file a new issue. Just would like to avoid things getting mixed up in a commit.

Will continue reviewing other change in PR.

Reply:
Yes, it is. Do you want me to put this into a STORM-3116 PR, or what do you mean?

Reply:
I hope to see this being separated PR, tagging as STORM-3116 or just tagging as MINOR. Either is fine.

Reply:
Moved to https://github.com/apache/storm/pull/2736